uk.co.compucorp.civicrm.appraisals/info.xml

```diff
@@ -8,8 +8,8 @@
     <author>coldrunKacper</author>
     <email>kacper@coldrun.pl</email>
```

Comment:
Do we need to standardise/unify the maintainer author and email across the CIVIHR extensions?